python-frontend/src/test/java/org/sonar/python/semantic/ProjectLevelSymbolTableTest.java

```diff
@@ -663,16 +660,20 @@ void class_symbol() {
     assertThat(cSymbol.name()).isEqualTo("C");
     assertThat(cSymbol.kind()).isEqualTo(Symbol.Kind.CLASS);
     assertThat(((ClassSymbol) cSymbol).superClasses()).hasSize(1);
+  }
 
+  @Test
+  @Disabled("SONARPY-2250")
```

Comment:
I'm not sure how critical is it, but this disabled test actually doesn't highlight what is wrong with the inheritance from a nested class, and it passes successfully, however, there is another problem - an incorrect superclass symbol fully qualified name

Reply:
Good catch! Let's merge this branch and have a look on that against master tomorrow.